examples/flutter_gallery/lib/gallery/themes.dart

```diff
@@ -43,6 +43,12 @@ ThemeData _buildDarkTheme() {
       colorScheme: colorScheme,
       textTheme: ButtonTextTheme.primary,
     ),
+    sliderTheme: SliderThemeData.fromPrimaryColors(
+      primaryColor: primaryColor,
+      primaryColorDark: primaryColor,
```

Comment:
For the time being, let's use secondaryColor for primaryColorLight, and Color(0xFF0050a0) for primaryColorDark